neo4j/neo4j.go

```diff
@@ -11,6 +13,44 @@ type Neo4jRuntime struct {
 	result  neo4j.Result
 }
 
+var (
+	Neo4jInjectionRegex = regexp.MustCompile("^(.*?)(?:(?:\"(?:.*)\")|(?:'(?:.*)'))(.*?)\\s*(?:" +
+		"(?:SET)|" +
+		"(?:CREATE)|" +
+		"(?:UPDATE)|" +
+		"(?:MATCH)|" +
+		"(?:RETURN)|" +
+		"(?:WITH)|" +
+		"(?:UNWIND)|" +
+		"(?:WHERE)|" +
+		"(?:EXISTS)|" +
+		"(?:ORDER BY)|" +
+		"(?:SKIP)|" +
+		"(?:LIMIT)|" +
+		"(?:USING)|" +
+		"(?:DELETE)|" +
+		"(?:DETACH)|" +
+		"(?:REMOVE)|" +
+		"(?:FOR EACH)|" +
+		"(?:MERGE)|" +
+		"(?:ON CREATE)|" +
+		"(?:ON MATCH)|" +
+		"(?:CALL)|" +
+		"(?:YIELD)|" +
+		"(?:USE)|" +
+		"(?:DROP)|" +
+		"(?:START)|" +
+		"(?:STOP)" +
+		")\\s(.*?)")
+)
+
+func (n *Neo4jRuntime) CheckForInjection(expStr string) bool {
```

Comment:
Also return the severity of injection. For example, if an injection can potentially wipe the data.